packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/utils/tests/select.test.js

```diff
@@ -138,7 +138,7 @@ describe('RelationInputDataManager | select', () => {
     const { result } = await setup(SELECT_ATTR_FIXTURE);
 
     expect(result.current.queryInfos.endpoints.relation).toBe(
-      '/content-manager/collection-types/slug/2/test'
+      '/content-manager/relations/slug/2/test'
```

Comment:
Could you add a test for `componentId` in here, please? :)

Reply:
sorry I'm not sure I got what tests you'd like 👀
I added 2 tests [here](https://github.com/strapi/strapi/pull/14544/files#diff-40d05a4906ced191514d9c965402456702f818ec369682d272305293bccd8453R186) to simulate relation endpoints  for `componentId` which should cover the use cases for single/rep comp and dz comp?

Reply:
added a Jira ticket to not forget this tests discussions